packages/block-editor/src/components/block-list/block.js

```diff
@@ -251,8 +252,9 @@ function BlockListBlock( {
 		}
 	}, [ isFirstMultiSelected ] );
 
+	const { isTyping } = useSelect( ( select ) => ( { isTyping: select( 'core/block-editor' ).isTyping() } ) );
```

Comment:
Somehow, this making all tests in `packages/e2e-tests/specs/navigable-toolbar.test.js` fail.